coderd/coderdtest/coderdtest.go

```diff
@@ -241,6 +244,22 @@ func NewOptions(t testing.TB, options *Options) (func(http.Handler), context.Can
 	if options.FilesRateLimit == 0 {
 		options.FilesRateLimit = -1
 	}
+	if options.StatsBatcher == nil {
+		batchStatsTicker := time.NewTicker(time.Hour)
+		t.Cleanup(batchStatsTicker.Stop)
+		options.StatsBatcher, err = batchstats.New(
+			batchstats.WithStore(options.Database),
+			// Some tests rely on being able to read stats immediately after writing.
+			batchstats.WithBatchSize(1),
```

Comment:
reivew: I'm explicitly setting a low batch size for unit tests here.
Alternatively, I can set to the default batch size and pass in a Batcher with a small batch size in the specific unit tests that need it. Thoughts?

Reply:
Which option will make unit tests execute faster?

Reply:
I expect the latter option, as there would be fewer DB calls overall. I'll go with that one then.